src/components/HeroSection.tsx

```diff
@@ -1,8 +1,8 @@
-"use client"
-import { Box, Button, Typography } from "@mui/material"
-import Image from "next/image"
-import CreditCardImage from "../assets/images/credit-card.webp"
-import CoinDeskLogo from "../assets/images/coin-desk-logo.png"
+"use client";
```

Comment:
Look like this file is using a different code format from the configuration
Can you try to run `npm run format` to format the code